integration_test/third_party_apps_data/agent/ops-agent/linux/enable_tomcat

```diff
@@ -9,7 +9,7 @@ sudo tee /etc/google-cloud-ops-agent/config.yaml > /dev/null << EOF
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
-#      http://www.apache.org/licenses/LICENSE-2.0
+#      http://www.tomcat.org/licenses/LICENSE-2.0
```

Comment:
sed replaced by accident? (Can we also double check the licenses in other files)

Reply:
whoops fixed, double checked others as well.